sdks/java/io/amazon-web-services/src/main/java/org/apache/beam/sdk/io/aws/options/AwsOptions.java

```diff
@@ -103,6 +103,34 @@ public AWSCredentialsProvider create(PipelineOptions options) {
 
   void setClientConfiguration(ClientConfiguration clientConfiguration);
```

Comment:
Are `setClientConfiguration` and `getClientConfiguration` still valid?

Reply:
yaa they could be gotten rid of but I wasn't sure if it was a breaking change for people already using it. If you think it's ok to have only one i will change it.

Reply:
Yes, you are right doesn't bring in any breaking change.

Reply:
I think it would be better to keep it for now to avoid breaking change. Perhaps, just add a comment that it's recommended to use new methods since they provide more functionality.

Reply:
Donee !! Added a note to use the newer one. 👍